src/parser/priest/shadow/modules/features/Buffs.tsx

```diff
@@ -15,14 +15,25 @@ class Buffs extends CoreBuffs {
         timelineHighlight: true,
       },
       {
-        spellId: SPELLS.DARK_THOUGHTS.id,
+        spellId: SPELLS.DARK_THOUGHT_BUFF.id,
```

Comment:
prolly should be "Dark Thought_s_"?

Reply:
The passive ability is "Dark Thoughts" and the proc is "Dark Thought" -- silly I know but those are the ability names